doc/bitcoin-conf.md

```diff
@@ -49,4 +49,12 @@ rpcport=4000
 
 The configuration file is not automatically created; you can create it using your favorite text editor. By default, the configuration file name is `bitcoin.conf` and it is located in the Bitcoin data directory, but both the Bitcoin data directory and the configuration file path may be changed using the `-datadir` and `-conf` command-line options.
 
+### Default configuration file locations
+
+| Operating System | Data Directory | Path |
```

Comment:
Please remove the `|` at the left border

Reply:
See doc/files.md

Reply:
I see. I copy-pasted a table and edited it from somewhere else in this repository.